src/fs_fd.js

```diff
@@ -6,7 +6,7 @@ import { Fd } from "./fd.js";
 
 export class OpenFile extends Fd {
     /*:: file: File*/;
-    file_pos/*: number*/ = 0;
+    file_pos/*: number*/ = 0n;
```

Comment:
Could you update the type hint to use BigInt?

Reply:
Not sure if my fix is correct, running flow locally seemingly picks up only errors in `fd.js` afterwards.